gen3/bin/kube-setup-tube.sh

```diff
@@ -9,7 +9,7 @@ source "${GEN3_HOME}/gen3/lib/utils.sh"
 gen3_load "gen3/lib/kube-setup-init"
 
 gen3 kube-setup-secrets
-gen3 roll tube
+gen3 roll spark $@
```

Comment:
is this supposed to be `gen3 roll tube ...` ?

Reply:
thanks